### PR TITLE
Add Jupyter Notebook with emcpy examples

### PR DESCRIPTION
**Pull request by kevindougherty-noaa**
This notebook produces visuals for all the existing test scripts to give users an idea of how to use emcpy.

### CODE REVIEW

File: src/tests/EMCPy_Examples_Notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,1144 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append('/scratch1/NCEPDEV/da/Kevin.Dougherty/emcpy/src/')"
```
Comment (CoryMartin-NOAA) [inline src/tests/EMCPy_Examples_Notebook.ipynb:30]: Probably should remove this before merging.

**Review by CoryMartin-NOAA — CHANGES_REQUESTED** [review]
I don't think the .ipynb files should be in the src/tests directory. Presumably there should be something like Examples/ at the same level as src/ ? What do things like matplotlib or pandas do?

**Comment by kevindougherty-noaa** [comment]
@CoryMartin-NOAA What do you mean by what do pandas and matplotlib do?

**Comment by CoryMartin-NOAA** [comment]
> @CoryMartin-NOAA What do you mean by what do pandas and matplotlib do?
https://github.com/matplotlib/matplotlib/tree/main/examples

**Comment by kevindougherty-noaa** [comment]
@CoryMartin-NOAA Ah I see, that is a much better idea. Then I think each example should be separated into individual descriptive directories as .py files, that way it's much easier for someone who needs a specific plot/example to find a base example of what they are looking for. I will close this PR and open a new one to better represent that.

**Comment by CoryMartin-NOAA** [comment]
@kevindougherty-noaa don't need to have each test as an example but perhaps just a couple of the plot ones for now.

**Pull request closed without merging**